commons/Common.mli

```diff
@@ -149,27 +247,24 @@ val hashset_of_list : 'a list -> 'a hashset
 val hashset_to_list : 'a hashset -> 'a list
 
 val optlist_to_list: 'a list option -> 'a list
-(* you should prefer let ( let* ) = Option.bind though *)
+
+(* you should prefer let ( let* ) = Option.bind though
```

Comment:
This comment is confusing to me. I read it as "you should prefer `let ( let* ) = Option.bind` though TODO: delete those functions, let* is better". Are they supposed to be separate comments that both suggest `let*`?